### DISCUSSION TITLE
Cant use heroicons in next

### DISCUSSION THREAD

**Discussion by sahuyash1103** [tailwindlabs/heroicons#776] [Help]
![image](https://user-images.githubusercontent.com/72812188/187035729-083bbc2c-3fc4-465d-80c1-850447088a4d.png)

I have tried every import, but none of them is working. I can not use any icon in next. But only the version 1.0.6 is working please help me what's wrong with v2.x

**Comment by adamwathan** [comment]
Hey, your import is not correct for v2. Should be `@heroicons/react/24/solid` or `20` if you want the smaller set 👍🏻

**Reply by sahuyash1103** [reply]
I tried that also but nothing happened

**Reply by sahuyash1103** [reply]
Using this  @heroicons/react/24/solid can't import icons, i.e. icons aren't there as of VS Code says
